torchao/quantization/quantize_/workflows/__init__.py

```diff
@@ -14,6 +14,9 @@
 from .int4.int4_tensor import (
     Int4Tensor,
 )
+from .int4.int4_xpu_tensor import (
+    Int4XPUTensorIntZP,
```

Comment:
nit: we should rename the tensor names etc. as well

Reply:
the packing format should contain a description of how the quantized weights are laid out in memory, it should not mention `XPU`, and also not `INTZP` since these are not relevant to how the quantized weights laid out.

can you change it to something like `Int4PlainInt32` since `PLAIN_INT32` is the packing_format?

Reply:
Int4PlainInt32 is good for me. But I still have some concern about the relation between packing_format and subclassing tensor? it must be 1 x 1?

Reply:
I think so, it's 1x1, we are also going to do a follow up refactor to have Int4PackingFormat instead having a global PackingFormat, since there is not much reuse

what's the concern for this?

Reply:
Thanks. Your explanation is clear enough. No more concern now.